modules/local/summarizeMultiqcPair/resources/usr/bin/summarize-multiqc-pair.R

```diff
@@ -13,11 +13,14 @@ option_list = list(
               help="Stage descriptor."),
   make_option(c("-S", "--sample"), type="character", default=NULL,
               help="Sample ID."),
+  make_option(c("-r", "--single_end"), type="character", default=FALSE,
```

Comment:
Shouldn't you make type here boolean if you're doing what you're converting it in the previous script. Noting that I don't know if this is an option for Rparse but I think it should be.

Reply:
Fixed!